s3tests_boto3/functional/test_s3.py

```diff
@@ -9160,6 +9776,34 @@ def test_bucket_policy():
     response = alt_client.list_objects(Bucket=bucket_name)
     eq(len(response['Contents']), 1)
 
+def test_bucketv2_policy():
```

Comment:
This test and all of the other bucket policy tests need `@attr` decorators.

Reply:
Added